text/0000-named-tracers.md

```diff
@@ -0,0 +1,74 @@
+# Named Tracers
+
+**Status:** `proposed`
+
+_Creating tracers using a factory mechanism and naming those tracers in accordance with the library that provides the instrumentation for a traced component._
+
+## Suggested reading
+
+* [Proposal: Tracer Components](https://github.com/open-telemetry/opentelemetry-specification/issues/10)
+* [Global Instance discussions](https://github.com/open-telemetry/opentelemetry-specification/labels/global%20instance)
+
+## Motivation
+
+Instead of having a global tracer singleton, every instrumentation library needs to create its own tracer, which also has a proper name that identifies the library / logical component that uses this tracer. If there are two different instrumentation libraries for the same technology (e.g. MongoDb), these instrumentation libraries should have distinct names. In this sense, this proposal suggests the introduction of a "semantic namespace" concept for tracers and the resulting spans.
+
+The purpose of _Named Tracers_ is the ability to allow for associating an identifier (e.g. _"io.opentelemetry.contrib.mongodb"_) with a tracer. This further allows for logical and semantic coupling of this tracer and a certain technology or library (like _MongoDB_ in this example). Having named tracers enables support for having a dedicated property (component) depending on this name on a tracer that is different for different libraries.
```

Comment:
I would leave out the sentence with the dedicated property. That is basically a restatement of "associating an indentifier". Also what is the difference between "logical" and "semantic" coupling? What does coupling even mean here? Is that another restatement of being able to associate an identifier?

I would rather like to see examples of what *useful* things (e.g. from the perspective of someone that deploys a traced system) can be achieved with that identifier. For example, ability of API-implementations to provide server-side disabling of certain tracers by name, adding an additional property to a Span to ease interpretation of attributes, etc.